acceptance/launcher_test.go

```diff
@@ -44,21 +47,28 @@ func TestLauncher(t *testing.T) {
 func testLauncher(t *testing.T, when spec.G, it spec.S) {
 	when("Buildpack API >= 0.5", func() {
 		when("exec.d", func() {
-			it.Before(func() {
-				h.SkipIf(t, runtime.GOOS == "windows", "exec.d is not yet supported on windows")
-			})
-
 			it("executes the binaries and modifies env before running profiles", func() {
 				cmd := exec.Command("docker", "run", "--rm",
 					"--env=VAR_FROM_EXEC_D=orig-val",
 					launchImage, "exec.d-checker")
-				expected := "/layers/0.5_buildpack/some_layer/exec.d/helper was executed\n"
-				expected += "Exec.d Working Dir: /workspace\n"
-				expected += "/layers/0.5_buildpack/some_layer/exec.d/exec.d-checker/helper was executed\n"
-				expected += "Exec.d Working Dir: /workspace\n"
+
+				helper := "helper" + exe
+				execDHelper := filepath.Join(rootDir, "layers", "0.5_buildpack", "some_layer", "exec.d", helper)
+				execDCheckerHelper := filepath.Join(rootDir, "layers", "0.5_buildpack", "some_layer", "exec.d", "exec.d-checker", helper)
+				workDir := filepath.Join(rootDir, "workspace")
+
+				expected := fmt.Sprintf("%s was executed\n", execDHelper)
+				expected += fmt.Sprintf("Exec.d Working Dir: %s\n", workDir)
+				expected += fmt.Sprintf("%s was executed\n", execDCheckerHelper)
+				expected += fmt.Sprintf("Exec.d Working Dir: %s\n", workDir)
 				expected += "sourced bp profile\n"
 				expected += "sourced app profile\n"
-				expected += "VAR_FROM_EXEC_D: orig-val:val-from-exec.d:val-from-exec.d-for-process-type-exec.d-checker"
+				if runtime.GOOS == "windows" {
+					expected += `"VAR_FROM_EXEC_D: orig-val:val-from-exec.d:val-from-exec.d-for-process-type-exec.d-checker"`
```

Comment:
What causes this difference? Would it make sense to lessen the assertion to a contains to remove this OS conditional you think?

Reply:
Sounds good... I think this is a difference in how `printf` on Linux and `echo` on Windows process args.